src/components/common.js

```diff
@@ -7,6 +7,7 @@ import { b, div, h, h1, img, input, label, span } from 'react-hyperscript-helper
 import RSelect, { components as RSelectComponents } from 'react-select'
 import RAsyncCreatableSelect from 'react-select/async-creatable'
 import RSwitch from 'react-switch'
+import { ButtonPrimary } from 'src/components/common/buttons'
```

Comment:
Why the explicit import here given the wildcard down at 34?

Reply:
Other components in this module (fence links, delete confirmation modal) use ButtonPrimary. This import is for those uses.